brendenwest/cascade
# Calendar Screen

**### Description**
Closes [#12 ](https://github.com/brendenwest/cascade/issues/12)


**### To Test**

1. git checkout `calendar-screen`
2. git pull
3. npm install
4. npx react-native start
5. [New terminal] npx react-native run-android
6. Make sure the app runs without errors!
7. Ensure ALL aspects of the app works correctly
